containers/docker/deployment/admin/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM hpccsystems/hpcc-base:7-bionic
+
+ARG version=
+RUN if [ -z "${version}" ] ; then echo Missing HPCC version; exit 1; else : ; fi
+
+#ENV PLATFORM_DOWNLOAD_MD5  a62b43a1ef1ecf7d8d702d9033e9cf39
+RUN set -ex; \
+   apt-get update; \
+   apt-get install -y iputils-ping curl xterm libarchive13 ansible iputils-ping python3-pip libssl-dev ; \
```

Comment:
We don't list all prerequisites for installation. Michael implements the auto-install prerequisites. But still need list here since Docker base image miss lots standard Linux distro packages